deploy/crds/alerts_v1alpha1_storagealerts_crd.yaml

```diff
@@ -0,0 +1,59 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: storagealerts.alerts.monstorak.org
+spec:
+  group: alerts.monstorak.org
+  names:
+    kind: StorageAlerts
+    listKind: StorageAlertsList
+    plural: storagealerts
+    singular: storagealerts
```

Comment:
plural and singular names, both same? Shouldn't we have singular as `storagealert` ?

Reply:
Intentionally kept the plural, the thought process behind was, we are showing multiple alerts not a single alert.

Reply:
I feel its more represents a Storage alerts class so lets rename as `StorageAlertsClass` and then we will have proper singular as `storagealertclass` and plural as `storagealertclasses`. Sorry I am making too many changes, but makes sense I feel :)

Reply:
@shtripat , as agreed, I will merge this PR and we will test our over all flow. We have a separate issue raised (and corresponding PR checked in) for the basic CRD design (issue:https://github.com/monstorak/monstorak/issues/6) . We will discuss the changes and finalize there. Once it is up to the mark, we can put the final changes to the code.

Reply:
PR tracking the issue: https://github.com/monstorak/monstorak/pull/19